.gitlab/jobs/corona.yml

```diff
@@ -27,9 +27,17 @@
 # ${PROJECT_<MACHINE>_DEPS} in the extra jobs. There is no reason not to fully
 # describe the spec here.
 
+clang_19_0_0_sycl_gcc_10_3_1_rocmcc_6_3_1_hip:
+  variables:
+    SPEC: " ~shared +sycl ~openmp tests=basic cxxflags==\"-w -fsycl -fsycl-unnamed-lambda -fsycl-targets=amdgcn-amd-amdhsa -Xsycl-target-backend --offload-arch=gfx906\" %sycl-clang-19-gcc-10 ^blt@develop"
+    MODULE_LIST: "rocm/6.3.1"
+    SYCL_PATH: "/usr/workspace/raja-dev/clang_sycl_e42590ef4b94_hip_gcc10.3.1_rocm6.3.1/"
+    LD_LIBRARY_PATH: "${SYCL_PATH}/lib:${SYCL_PATH}/lib64:${LD_LIBRARY_PATH}"
+  extends: .job_on_corona
+
 clang_20_0_0_sycl_gcc_10_3_1_rocmcc_6_0_2_hip:
   variables:
-    SPEC: " ~shared +sycl ~openmp tests=basic %clang@=20.0.0 cxxflags==\"-w -fsycl -fsycl-unnamed-lambda -fsycl-targets=amdgcn-amd-amdhsa -Xsycl-target-backend --offload-arch=gfx906\" ^blt@develop"
+    SPEC: " ~shared +sycl ~openmp tests=basic cxxflags==\"-w -fsycl -fsycl-unnamed-lambda -fsycl-targets=amdgcn-amd-amdhsa -Xsycl-target-backend --offload-arch=gfx906\" %sycl-clang-20-gcc-10 ^blt@develop"
```

Comment:
This job can be removed. The new one above (rocm 6.3.1) supersedes. Removing this may fix the timeout issue.

Reply:
Please proceed. Deleting this section should suffice.